modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -4,10 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 
 
-Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine, such as policies, contracts, and the credentials required to configure the connection between your Mule application and Anypoint Platform.
-
-This feature is supported in Mule 4.2.0 and later.
+Gateway startup encryption enables you to encrypt sensitive information stored by Mule runtime engine (Mule), such as policies, contracts, and the credentials required to configure the connection between your Mule application and Anypoint Platform.
 
+This feature is supported in Mule 4.2.0 and later. If a <<policies-that-support-encryption, policy that supports encryption>>, such as Basic Authentication 1.2.0, is applied to an API that is running in Mule versions earlier than 4.2.0, the policy is not encrypted.
```

Comment:
Forgive my ignorance, but is a space allowed after the comma that defines the anchor in a <<anchor,text>> sequence? I always see it as not having a space, not sure if it works with one.

Reply:
@Mulesoft-Mysti It does work and this is how I have used it. Let me know if I'm doing it all wrong.